LL/de_rigaib_muman_iar_cretim_inso.ttl

```diff
@@ -0,0 +1,580 @@
+@base <http://example.com/LL/de_rigaib_muman_iar_cretim_inso.ttl>.
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix foaf:  <http://xmlns.com/foaf/0.1/> .
+@prefix rel: <http://purl.org/vocab/relationship/>.
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix dctype: <http://purl.org/dc/dcmitype/> .
+@prefix irishTitles: <http://example.com/earlyIrishTitles.ttl#> .
+
+<>
+    a dctype:Dataset;
+    dcterms:title "De Rigaib Muman Iar Cretim Inso"@sga;
+    dcterms:isFormatOf <http://www.ucc.ie/celt/published/G800011F/text006.html>;
+    dcterms:format "text/turtle".
+
+<#Oengus>
+    a irishTitles:Rí ;
+    irishTitles:reignLength 36 ;
+    rel:childOf <#NathFraich>;
+    irishTitles:succeededBy <#Eocho>;
+    foaf:name "Oengus" ;
+    owl:sameAs <http://example.com/LL/eognachta_casil.ttl#Oengusa>.
+
+<#NathFraich>
+    owl:sameAs <http://example.com/LL/eognachta_casil.ttl#NathFraich-47cc38b6>.
+
+<#Eocho>
+    a irishTitles:Rí ;
+    irishTitles:succeededBy <#Feidlimid>;
+    rel:childOf <#Oengusa>;
+    foaf:name "Eocho";
+    irishTitles:reignEndedBy "marb"@sga .
+    
+<#Oengusa>
+    a foaf:Person ;
+    foaf:name "Oengusa" .
+
+<#Feidlimid>
```

Comment:
Chronologically, it would make sense for this to be <http://example.com/LL/eoganachta_casil.ttl#Feidlimid-ffa55005> (he would be Eocho mac Oengussa's brother). But there is no direct evidence.